pills/02-install-on-your-running.xml

```diff
@@ -124,7 +124,11 @@
       xlink:href="https://nixos.org/manual/nix/stable/package-management/profiles.html">profile</link>:
     </para>
 
-    <xi:include href="./02/user-environment.xml" parse="xml" />
+    &lt;screen&gt;creating /home/nix/.nix-profile
```

Comment:
You are accidentally escaping the `<screen>` tags here.

Reply:
Fixed.